posts/2024-10-22-24.0.0.11-beta.adoc

```diff
@@ -0,0 +1,168 @@
+
+* <<instant, Use Liberty InstantOn with authorization and messaging features>>
+* <<class, Configure the web module class loader for an enterprise application>>
+
+See also link:{url-prefix}/blog/?search=beta&key=tag[previous Open Liberty beta blog posts].
+
+// // // // DO NOT MODIFY THIS COMMENT BLOCK <GHA-BLOG-TOPIC> // // // //
+// Blog issue: https://github.com/OpenLiberty/open-liberty/issues/29768
+// Contact/Reviewer: anjumfatima90
+// // // // // // // //
+
+[#instant]
+== Use Liberty InstantOn with authorization and messaging features
+
+InstantOn supports link:{url-prefix}/docs/latest/instanton.html#supported-features[a subset of Open Liberty features]. As of the 24.0.0.11-beta release, the following features are enhanced to support InstantOn:
+
+* link:{url-prefix}/docs/latest/reference/feature/jacc-1.5.html[Java Authorization Contract for Containers (jacc-1.5)].
+* link:{url-prefix}/docs/latest/reference/feature/appAuthorization-2.0.html[Jakarta Authorization 2.0 (appAuthorization-2.0)]
+* link:{url-prefix}/docs/latest/reference/feature/appAuthorization-2.1.html[Jakarta Authorization 2.1  (appAuthorization-2.1)]
+* link:{url-prefix}/docs/latest/reference/feature/wasJmsServer-1.0.html[Message Server 1.0 (wasJmsServer-1.0)]
+* link:{url-prefix}/docs/latest/reference/feature/messagingServer-3.0.html[Messaging Server 3.0 (messagingServer-3.0.html)]
+* link:{url-prefix}/docs/latest/reference/feature/wasJmsClient-2.0.html[JMS Client for Message Server 2.0 (wasJmsClient-2.0)]
+* link:{url-prefix}/docs/latest/reference/feature/messagingClient-3.0.html[Messaging Server Client (messagingClient-3.0.html)]
+
+This enhancement enables you to rapidly deploy applications that use these features by using Open Liberty InstantOn. For more information, see link:{url-prefix}/docs/latest/instanton.html[Faster startup for containerized applications with Open Liberty InstantOn].
+
+
+// DO NOT MODIFY THIS LINE. </GHA-BLOG-TOPIC>
+
+// // // // DO NOT MODIFY THIS COMMENT BLOCK <GHA-BLOG-TOPIC> // // // //
+// Blog issue: https://github.com/OpenLiberty/open-liberty/issues/29766
+// Contact/Reviewer: tjwatson
+// // // // // // // //
+[#class]
+== Configure the web module class loader for an enterprise application
+
+Web modules (WARs) that are part of an enterprise application (EAR) can specify their own link:https://docs.oracle.com/javase/8/docs/technotes/guides/jar/jar.html#classpath[Class-Path]. By default, Liberty adds the JARs that are referenced by a web module `Class-Path` attribute to the web module class loader. However, some runtime implementations of Java EE and Jakarta EE, such as traditional WebSphere, instead use the enterprise application class loader for these JARs. Typically, the enterprise application class loader is the parent class loader for the web module class loader.
```

Comment:
The use of "traditional WebSphere" should be changed to "WebSphere Application Server traditional".